src/components/css/FilterBoard.css

```diff
@@ -1,16 +1,27 @@
-.FilterButton{
-    border-radius: 6px;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    margin: 0 2px;
-    background: #fc0303;
-  }
+.FilterButton {
+  border-radius: 6px;
+  display: flex;
+  flex-direction: row;
```

Comment:
You don't need `flex-direction: row`: It's the default value, if you're using `display: flex`.
Saw this on another position as well, but was not part of the pull request. But worth to change.